Source/Moya.swift

```diff
@@ -150,6 +150,12 @@ public extension MoyaProvider {
     public final class func DefaultRequestMapping(endpoint: Endpoint<Target>, closure: NSURLRequest -> Void) {
         return closure(endpoint.urlRequest)
     }
+
+    public final class func DefaultAlamofireManager() -> Manager {
+        let manager = Alamofire.Manager.sharedInstance
+        manager.startRequestsImmediately = false
```

Comment:
Hmm, I'm not in love with the idea of changing Alamofire's singleton's state, but I'm not sure I see another option.